rules/opentitan.bzl

```diff
@@ -470,7 +545,10 @@ def opentitan_rom_binary(
     for (device, dev_deps) in per_device_deps.items():
         devname = "{}_{}".format(name, device)
 
-        # Generate ELF, Binary, and Disassembly
+        # Generate ELF, Binary, Disassembly, and (maybe) sim_dv logs database
+        kwargs["extract_sw_logs_db"] = False
+        if device == "sim_dv":
+            kwargs["extract_sw_logs_db"] = True
```

Comment:
You probably could avoid playing with kwargs here like this:
```
        targets.extend(opentitan_binary(
            name = devname,
            deps = deps + dev_deps,
            extract_sw_logs_db = device == "sim_dv",
            **kwargs
        ))
```

Reply:
ahh right, it's python, nice!

fixed.